orttraining/orttraining/test/python/orttraining_ortmodule_distributed_tests.py

```diff
@@ -33,6 +33,22 @@ def run_ortmodule_deepspeed_zero_stage_1_tests(cwd, log, data_dir):
 
     run_subprocess(command, cwd=cwd, log=log).check_returncode()
 
+def run_ortmodule_deepspeed_pipeline_parallel_tests(cwd, log):
+    log.debug('Running: ORTModule deepspeed pipeline parallel tests')
+
+    command = ['deepspeed', 'orttraining_test_ortmodule_deepspeed_pipeline_parallel.py',
+        '--deepspeed_config', 'orttraining_test_ortmodule_deepspeed_pipeline_parallel_config.json']
+
+    run_subprocess(command, cwd=cwd, log=log).check_returncode()
+
+def run_ortmodule_fairscale_sharded_optimizer_tests(cwd, log, data_dir):
+    log.debug('Running: ORTModule fairscale sharded optimizer tests')
+    command = ['python3', 'orttraining_test_ortmodule_fairscale_sharded_optimizer.py',
```

Comment:
We could use sys.executable instead. So that it runs with the same python version as the current file is being run in.

Reply:
Can address this in the next PR.